lib/modules/ens/embarkjs.js

```diff
@@ -298,7 +187,9 @@ __embarkENS.lookup = function(address) {
     let resolverContract = new web3.eth.Contract(self.resolverInterface, resolverAddress);
     return resolverContract.methods.name(node).call();
   }).then((name) => {
-    if (name === "" || name === undefined) throw Error("ENS name not found");
-    return name;
-  }).catch(err => err);
+    if (name === "" || name === undefined) {
+      cb(new Error("Name not found"), null);      
```

Comment:
No need to pass null as second parameter. Putting nothing sets it as undefined, which is the same in terms of truthiness